src/main.rs

```diff
@@ -302,6 +329,20 @@ fn app() -> clap::App<'static, 'static> {
         .arg(Arg::with_name("update-index")
              .long("update-index")
              .help("Forcefully re-indexes the IMDb data and then exits."))
+        .arg(Arg::with_name("symlink")
+             .long("symlink")
+             .short("s")
+             .value_name("folder")
+             .conflicts_with("hardlink")
+             .help("Place symlink in <folder> instead of moving. Be careful of \
+                   absolute vs. relative links: the target of the symlink will \
+                   match the given input. (Unix only feature)"))
```

Comment:
I'm having trouble figuring out why `symlink` takes an argument here. Wouldn't a destination directory be more appropriate as a separate standalone flag? It seems like an orthogonal concept to symlinks...

Reply:
I agree. I'll add a `--destination` / `-d` Arg. this would also apply to normal renames. the file would be renamed, and moved to the destination (only if specified of course).
in the situation `symlink` is specified but no destination is specified, a symlinked file will appear next to the original file in the same directory.

Reply:
@BurntSushi because of absolute/relative paths concerning symlinks, these are the behaviors I imagine being useful defaults:

| Action | no destination specified | `-d outdir` specified |
| --- | --- | --- |
| rename | file is renamed in place (current behavior) | file is moved to `outdir` after being renamed. In code this would still be a single `std::fs::rename` |
| symlink | well-named link appears in current directory and points to the original filepath, however given.<br><br>If the original path is relative (`imdb-rename -s ./movies/Movie.2010.1080p.mkv`) then the link will be relative and only work if the symlink remains in the current directory: `Movie (2010).mkv -> ./movies/Movie.2010.1080p.mkv`.<br><br>If the path given is absolute (`imdb-rename -s /home/evan/movies/Movie.2010.1080p.mkv`) then the link will be absolute, and will work as expected even if moved from the current location. | well-named link appears in the folder `outdir` as requested, and will point to the absolute path of the files given, no matter how the files were specified on the commandline. This is the only way that the created symlink would know how to actually find the file in question without a lot more work, and is probably what the user expects anyway. |
| hardlink | named hardlink appears in current directory, and links correctly to original file regardless of filepath (just because of how hardlinks work) | well-named hardlink appears in `outdir`, and links correctly to original file regardless of filepath (just because of how hardlinks work) |

Reply:
alright this matrix as is, is implemented in the pr now